lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -51,12 +51,6 @@
         description:
             - Name of the newly deployed guest
         required: True
-   name_match:
-        description:
-            - If multiple vms matching the name, use the first or last found
-        required: False
-        default: 'first'
-        choices: ['first', 'last']
```

Comment:
to be clear, neither the docs NOR the functionality should be removed, i just posted one comment instead of 2 for the same issue.